funcoeszz

```diff
@@ -708,11 +709,11 @@ zzzz ()
 			local cod_sistema='ISO-8859-1'
 			local cod_funcoeszz='ISO-8859-1'
 
-			printf 'Verificando a codificação do sistema... '
+			printf '%-32s' '"Coding" do sistema... '
```

Comment:
Por que esta alteração? Achei pior em inglês.

Mas se for alterar, tem que atualizar o testador da `zzzz` também, e também o FAQ do site, que mostra essa mensagem.

Reply:
Acho porque testei em máquinas diferentes com SO's diferentes e estava tentando alinhar todos os resultados, é algo estético apenas mas totalmente  dispensável.
Vou retirar isso e deixar como estava.